web/components/PetPostModal/PetPostModal.tsx

```diff
@@ -0,0 +1,608 @@
+                        marginRight={2}
+                        marginTop={1}
+                        marginBottom={1}
+                      >
+                        <CheckIcon />
+                        <Text>Cats</Text>
+                      </Stack>
+                    </Center>
+                  </WrapItem>
+                  <WrapItem>
+                    <Center bgColor="#C6E3F9" borderRadius="25px">
+                      <Stack
+                        direction="row"
+                        alignItems="center"
+                        marginLeft={2}
+                        marginRight={2}
+                        marginTop={1}
+                        marginBottom={1}
+                      >
+                        <CheckIcon />
+                        <Text>Young Children</Text>
+                      </Stack>
+                    </Center>
+                  </WrapItem>
+                </Wrap>
+              </Stack>
+              <Stack direction="column">
+                <Text fontWeight="bold" fontSize="xl" fontFamily="sans-serif">
+                  I&apos;m not comfortable with
+                </Text>
+                <Wrap>
+                  <WrapItem>
+                    <Center bgColor="#C6E3F9" borderRadius="25px">
+                      <Stack
+                        direction="row"
+                        alignItems="center"
+                        marginLeft={2}
+                        marginRight={2}
+                        marginTop={1}
+                        marginBottom={1}
+                      >
+                        <CheckIcon />
+                        <Text>Cats</Text>
+                      </Stack>
+                    </Center>
+                  </WrapItem>
+                  <WrapItem>
+                    <Center bgColor="#C6E3F9" borderRadius="25px">
+                      <Stack
+                        direction="row"
+                        alignItems="center"
+                        marginLeft={2}
+                        marginRight={2}
+                        marginTop={1}
+                        marginBottom={1}
+                      >
+                        <CheckIcon />
+                        <Text>Young Children</Text>
+                      </Stack>
+                    </Center>
+                  </WrapItem>
+                </Wrap>
+              </Stack>
+            </Stack>
+          </Stack>
+          <Stack
+            width="100%"
+            justifyContent="center"
+            direction="row"
+            position="sticky"
+            bottom={0}
+            padding={4}
+            bgColor="white"
+          >
+            <Button
+              bgColor="angelsBlue.100"
+              color="white"
+              fontWeight="medium"
+              borderRadius="20px"
+              paddingLeft={40}
+              paddingRight={40}
+            >
+              Foster Me!
+            </Button>
+          </Stack>
+        </Stack>
```

Comment:
Button runs off the modal when the shrinking window width. Also consider using flexbox to position this element.